common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -243,10 +245,46 @@ class ModuleI18nService(object):
     i18n service.
 
     """
+    def __init__(self, block=None):
+        """
+        We are using this block for things like locating the block's translation domain (PO+MO files). Here we attempts
+         to look up the provided string in the XBlock's own domain translation catalog, currently expected to be:
+            <xblock_root>/conf/locale/<language>/LC_MESSAGES/<domain>.po
+        If ModuleI18nService can't locate the domain translation catalog then we fall-back onto
+        django.utils.translation.ugettext, which will attempt to find a matching string in the
+        LMS' own domain translation catalog -- effectively achieving translation by coincidence.
+        """
+        self.translator = django.utils.translation
+        if block:
+            xblock_resource = block.__class__.unmixed_class.__module__
```

Comment:
When you access attributes on an instance, if the instance doesn't have it, the class will be examined for it.  This is how self.method_foo is found.  You don't need the `__class__` here.